src/Client.php

```diff
@@ -397,7 +397,7 @@ private static function getVersion(): string
         if (self::$version === null) {
             try {
                 self::$version = InstalledVersions::getPrettyVersion('mongodb/mongodb') ?? 'unknown';
-            } catch (Throwable $t) {
+            } catch (Throwable) {
```

Comment:
Noted that the variable is no longer required as of PHP 8.0 (see: [Exceptions[(https://www.php.net/manual/en/language.exceptions.php)).